app/models/user.rb

```diff
@@ -4,7 +4,15 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
- def username
-    self.first_name + " " + self.last_name
- end
+  enum role: {client: 0, admin: 1}
```

Comment:
Use enum as a string like this

`ROLES = {
    client: 'client',
    admin: 'admin'
  }.freeze

  enum role: ROLES`